server.py

```
@@ -342,195 +549,341 @@ async def create_time_entry(
-            return {"success": False, "error": "End time must be after start time"}
-        
-        duration = int((end_dt - start_dt).total_seconds())
-        
-        # Create entry
-        data = {
-            "task_id": task_id,
-            "date": date,
-            "start_time": start_time + ":00",
-            "end_time": end_time + ":00",
-            "duration": duration
+        date_obj = datetime.strptime(request.date, "%Y-%m-%d")
```

Ruff (0.11.9)

Local variable `date_obj` is assigned to but never used

Remove assignment to unused variable `date_obj`

(F841)